### PR TITLE
migrate from cmd to well, add Go module support, migrate to circleci

### PR DESCRIPTION
**Pull request by Hsn723**
Replace cybozu-go/cmd with cybozy-go/well
Run Go mod init
Add CircleCI config and remove TravisCI config

### CODE REVIEW

**Comment by takahisa** [comment]
please `git commit --amend --reset-author`

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 [![GoDoc](https://godoc.org/github.com/cybozu-go/kkok?status.svg)][godoc]
 [![Build Status](https://travis-ci.org/cybozu-go/kkok.svg?branch=master)](https://travis-ci.org/cybozu-go/kkok)
```
Comment (ymmt2005) [inline README.md]: Replace this with the budge for CircleCI.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hsn723) [reply README.md]: Ok

**Review by ymmt2005 — APPROVED** [review]

**Comment by ymmt2005** [comment]
LGTM.  Thank you!

**Pull request merged**